lib/elixir/lib/string.ex

```diff
@@ -241,7 +241,8 @@ defmodule String do
   @doc """
   Divides a string into substrings at each Unicode whitespace
   occurrence with leading and trailing whitespace ignored. Groups
-  of whitespace are treated as a single occurrence.
+  of whitespace are treated as a single occurrence. Divisions do
+  not occur on whitespace marked as no-break by unicode.
```

Comment:
Don't forget to add `~S` to the sigil above to ensure `\u....` in the example below won't be escaped.